go-to/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I think the typings dependency is handled at the top level, so you should be able to remove `package.json` and `package-lock.json` from here.